config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"github.com/companieshouse/chs.go/log"
+	"github.com/ian-kent/gofigure"
+)
+
+// Config holds the session handler configuration
+type Config struct {
+	gofigure          interface{} `order:"env,flag"`
+	DefaultExpiration string      `env:"DEFAULT_EXPIRATION"		flag:"default-expiration"   flagDesc:"Default Expiration"`
+	CookieName        string      `env:"COOKIE_NAME"					flag:"cookie-name"          flagDesc:"Cookie Name"`
+	CookieSecret      string      `env:"COOKIE_SECRET"				flag:"cookie-secret"        flagDesc:"Cookie Secret"`
+	RedisServer       string      `env:"REDIS_SERVER"					flag:"redis-server"        	flagDesc:"Redis Server"`
+	RedisDB           int         `env:"REDIS_DB"							flag:"redis-db"       			flagDesc:"Redis DB"`
+	RedisPassword     string      `env:"REDIS_PASSWORD"				flag:"redis-password"       flagDesc:"Redis Password"`
+}
+
+var cfg *Config
+
+// Get returns a populated Config struct
+func Get() *Config {
+
+	if cfg != nil {
+		return cfg
+	}
+
+	cfg = &Config{}
+
+	if err := gofigure.Gofigure(cfg); err != nil {
```

Comment:
I may be wrong but I seem to remember that using `gofigure`  for configs in a library does not work as it will clash with the potential usage of `gofigure` in the service that depends on the library.

Reply:
I haven't encountered any problems yet, though that's not definitive proof that it works all the time. I spoke to @zxchris and he believes this problem was addressed. What's the best approach here?

Reply:
If we've got a project using this library that also uses gofigure to manage it's own configuration then we just need to check variations of it's config and this config to make sure both uses of gofigure are fully functional

Reply:
The `go-session-handler` and `go-sdk-manager` have been integrated with `document-request.web`, all of which use the Gofigure library. No problems as yet, but I'll make an addition to the `README` to make sure there are no errors going forward.